docs/kubernetes/ingress-usage.md

```diff
@@ -141,8 +141,41 @@ This example shows how to add predicates and filters:
 
 ## Custom Routes
 
-Custom routes is a way of extending the default routes configured for an
-ingress resource.
+Custom routes is a way of extending the default routes configured for
+an ingress resource.
+
+Sometimes you just want to return a header, redirect or even static
+html content. You can return from skipper without doing a proxy call
+to a backend, if you end your filter chain with `<shunt>`. The use of
```

Comment:
"The use of
+`<shunt>` requires, that you also use the `status()` filter."

i would phrase this such that "is recommended", because the <shunt> sets a 404 by default.